spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/data/parquet/vectorized/TestParquetVectorizedReads.java

```diff
@@ -117,6 +122,26 @@ private void writeAndValidate(
         setAndCheckArrowValidityVector,
         reuseContainers,
         batchSize);
+
+    // With encryption
+    testFile.delete();
+    try (FileAppender<GenericData.Record> writer = getEncryptedParquetWriter(schema, testFile)) {
+      writer.addAll(expected);
+    }
+    Assert.assertThrows(
+        "Decrypted without keys",
+        ParquetCryptoRuntimeException.class,
+        () ->
+            Parquet.read(Files.localInput(testFile)).project(schema).callInit().build().iterator());
```

Comment:
This should not use `callInit` because that is related to `ReadSupport`. I don't think this assertion is necessary anyway, so you should probably remove it.

Reply:
Ok, removing this assertion. Another type of encryption verification is done later in this flow.